client/js/app.modules/main/main.controller.js

```diff
@@ -1,7 +1,6 @@
 /**
  * @ngInject
  * */
-module.exports = function ($scope, $http, mainPageInfo){
-    $scope.isNavHide = true;
-    $scope.mainPageInfo = mainPageInfo;
+module.exports = function ($scope, $http, GlobalData){
+    this.GlobalData = GlobalData;
```

Comment:
Same: Use lowercase for properties ("globalData"). But its better to show only particular methods on scope (not whole service).